offchain/danaswap-api/test/Main.purs

```diff
@@ -36,16 +37,55 @@ main = launchAff_ $ do
     Nothing -> throw "expected MODE to be set"
   log "about to start"
   runWithMode mode $ do
-    describe "protocol init" $ do
+    describe "Liquidity Token Minting Policy" $ do
+
+      -- TODO there are more liquidity tests but
+      -- they depend partially on the pool address validator as well
+      -- so they will be added in a later PR
+
+      it "Allows minting on pool open" $ useRunnerSimple $ do
+        protocol <- initProtocol
+        openPool protocol
+
+      describe "Minting tokens for a different pool on pool open" $ do
+        it "fails to validate with the wrong redeemer" $ useRunnerSimple $ do
+          protocol <- initProtocol
+          expectError $ openPoolWrongTokenWrongRedeemer protocol
+
+        it "fails to validate with the right redeemer" $ useRunnerSimple $ do
```

Comment:
These still need capitalization.

Reply:
Sorry about that not sure how I missed those